packages/babel-plugin-proposal-decorators/package.json

```diff
@@ -22,7 +22,9 @@
   "dependencies": {
     "@babel/helper-create-class-features-plugin": "workspace:^",
     "@babel/helper-plugin-utils": "workspace:^",
-    "@babel/plugin-syntax-decorators": "workspace:^"
+    "@babel/helper-replace-supers": "workspace:^",
+    "@babel/plugin-syntax-decorators": "workspace:^",
+    "charcodes": "^0.2.0"
```

Comment:
This can be moved to `devDependencies` if we are going to inline it at compile time.